### ISSUE TITLE
Fix cloud tests timing out on travis

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/hellbender#750]
The cloud tests are timing out after 10 minutes without emitting any output.  It seems like `ApplyBQSRDataflowIntegrationTest.testPR_Cloud` is responsible.  It looks like something is crashing in dataflow but the runner is never stopped so it keeps waiting indefinitely (or at least 10 minutes..)  See the dataflow log [here](https://console.developers.google.com/project/broad-dsde-dev/dataflow/job/2015-07-24_12_44_26-17415749601435236766).   

Executing locally also seems to hang forever, with messages like 

```
Error:   (b65a2091061bf0f9): Workflow failed. Causes: (71540087aac21e37): Unable to create VMs. Causes: (71540087aac21994): Error:
Test: Test method testPR_Cloud[0](ApplyBQSR(args=''))(org.broadinstitute.hellbender.tools.walkers.bqsr.ApplyBQSRDataflowIntegrationTest) produced standard out/err:  Message: Value for field 'resource.metadata.items[1].value' is too large
```

Seems like this is possibly a dataflow bug.  If the workflow fails in some way the client should be released.

**Comment by lbergelson** [comment]
Update:  It does finish sometime.  It's just VERY SLOW.

**Comment by lbergelson** [comment]
@davidaadams I think the stalling is a dataflow bug.  We should make sure it's not fixed by #754, but I think we'll have to talk to the dataflow team about it if it's now.

**Comment by lbergelson** [comment]
This seems to have been fixed by #754.  The tests no longer time out, but they still fail (see #791.)
